### PR TITLE
Adding Peco v0.5.11

### PR DESCRIPTION
**Pull request by eunos-1128**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by eunos-1128** [comment]
@conda-forge/help-go

**Comment by eunos-1128** [comment]
> For recipes/peco/meta.yaml:
> 
> Please do not delete the example recipe found in recipes/example/recipe.yaml.

I've never edit recipes/example/recipe.yaml. conda-forge-linter seems to have any bugs.

**Comment by eunos-1128** [comment]
@conda-forge/help-go, ready for review!

**Review by xhochy — CHANGES_REQUESTED** [review]
Please aggregate third-party licenses using `go-licenses`

File: recipes/peco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8e32c8af533e03795f27feb4ee134960611d2fc0266528b1c512a6f1f065b164
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('go') }}
```
Comment (xhochy) [inline recipes/peco/meta.yaml:19]: `compiler('go')` is outdated and should no longer be used.

```suggestion
    - {{ compiler('go-nocgo') }}
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/peco/meta.yaml (hunk 1 above)
Comment (eunos-1128) [reply recipes/peco/meta.yaml]: Thanks, I committed it like you revised.

**Review by eunos-1128 — COMMENTED** [review]
Resolved.

**Review by eunos-1128 — COMMENTED** [review]
Resolved.

**Comment by eunos-1128** [comment]
@xhochy 

Sorry, I requested you to review again accidentally.

Please merge without worrying about it.

**Review by xhochy — APPROVED** [review]

**Pull request merged**